gsms.py

```diff
@@ -205,5 +304,38 @@ def main() -> None:
                                 'custom location.')
 
 
+def get_win_path(folderid, user_handle=UserHandle.current):
+    """
+    Function to resolve Windows UUID folders into their absolute path
+
+    Parameters
+    -------
+    folderid: string
+    user_handle: UserHandle
+
+    Returns
+    -------
+    Path as string
+
+    Raises
+    ------
+    PathNotFoundException
+        When a UUID can not be resolved to a path as it is not a windows UUID path
+
+    Examples
+    --------
+    >>> get_win_path("{B4BFCC3A-DB2C-424C-B029-7FE99A87C641}", UserHandle.Common)
+    C:\\Users\\Default\\Desktop
```

Comment:
```py
>>> get_win_path(folder_id="{B4BFCC3A-DB2C-424C-B029-7FE99A87C641}", user_handle=wintypes.HANDLE(0))
C:\\...
```

`...` represents wildcard match... technically `C:\\` shouldn't be there either as it might not be `C:\\` but it's fine since we're not testing the examples in doc strings yet.

Reply:
Changed to pure `...` wildcard